src/tests/suite/daos_obj.c

```diff
@@ -3223,8 +3223,9 @@ tgt_idx_change_retry(void **state)
 
 	if (arg->myrank == 0) {
 		print_message("rank 0 adding target rank %u ...\n", rank);
```

Comment:
minor, may be worth changing the print_message to say "adding rank" rather than "adding target rank"

Reply:
Done.